test/common/testlib.py

```diff
@@ -266,6 +266,7 @@ def select_from_dropdown(self, selector, value, substring=False):
         self.wait_present(text_selector)
         value_id = self.attr(text_selector, "value")
         self.set_val(selector, value_id)
+        self.wait_val(selector, value_id)
```

Comment:
Thanks! I think that was actually an omission/regression of mine from commit 342306445ffb3. The old implementation waited for the value, the new one didn't, so we didn't wait any more for the React handler to finish.